### PR TITLE
Merge in upgrades from the template

### PR DESCRIPTION
**Pull request by carver**
## What was wrong?

Follow-up to #17 

## How was it fixed?

Merged in template, as described here: https://github.com/ethereum/snake-charmers-tactical-manual/pull/62

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 The eth-typing contributors
+Copyright (c) 2019 The Ethereum Foundation
```
Comment (cburgdorf) [inline LICENSE]: Nitpick: I find  *`<year>` The `<project-name>` contributors*
Slightly more inclusive and hence welcoming to non EF contributors. But maybe this has legal implications that I'm not aware of.

File: README.md [hunk 2]
```diff
@@ -86,7 +86,11 @@ The version format for this repo is `{major}.{minor}.{patch}` for stable, and
 `{major}.{minor}.{patch}-{stage}.{devnum}` for unstable (`stage` can be alpha or beta).
 
 To issue the next version in line, specify which part to bump,
-like `make release bump=minor` or `make release bump=devnum`.
+like `make release bump=minor` or `make release bump=devnum`. This is typically done from the
+master branch, except when releasing a beta (in which case the beta is released from master,
+and the previous stable branch is released from said branch). To include changes made with each
+release, update "docs/releases.rst" with the changes, and apply commit directly to master
```
Comment (cburgdorf) [inline README.md]: Nothing about this PR but it reminds of the fact that we haven't yet merged the `towncrier` changes upstream: https://github.com/ethereum/ethereum-python-project-template/pull/28

We were holding it back to iron out issues in the release script.

By now we can probably sync it with the latest release script from Trinity and then get that in.

File: setup.py [hunk 3]
```diff
@@ -30,24 +31,26 @@
...
 setup(
     name='eth-typing',
     # *IMPORTANT*: Don't manually change the version here. Use `make bump`, as described in readme
     version='2.2.0',
     description="""eth-typing: Common type annotations for ethereum python packages""",
     long_description=long_description,
     long_description_content_type='text/markdown',
-    author='The eth-typing contributors',
-    author_email='eth-typing@ethereum.org',
+    author='The Ethereum Foundation',
```
Comment (cburgdorf) [inline setup.py]: Same here, not a strong stance, just a slight preference of using *The `<project-name>` authors*

**Comment by cburgdorf** [comment]
>Looks good except for the merge conflicts that Github is reporting.

Ah, scratch that. I guess that's just because my UI defaults to *Rebase and merge* (I like linear histories) which can not be used for this type of change (which is ok)

image

So, when I flip to just "Create a merge commit" it does look ok.

image

File: LICENSE (hunk 1 above)
Comment (carver) [reply LICENSE]: I don't have a strong feeling about one or the other. The inclusiveness argument has merit to me. But my strongest feeling is that we should have consistency.

I would probably approve a PR to the templates repo that changes this to "The <PROJECT_NAME> contributors". :)

File: README.md (hunk 2 above)
Comment (carver) [reply README.md]: Yeah, if the next trinity release is smooth with towncrier, I'm game to merge it to the template!

File: LICENSE (hunk 1 above)
Comment (cburgdorf) [reply LICENSE]: >I would probably approve a PR to the templates repo that changes this to "The <PROJECT_NAME> contributors". :)

I take your word for it.

https://github.com/ethereum/ethereum-python-project-template/pull/29

**Pull request merged**